processor-api/src/main/java/com/synopsys/integration/alert/processor/api/filter/FilteredJobNotificationWrapper.java

```diff
@@ -11,10 +11,12 @@
 import java.util.UUID;
 
 import com.synopsys.integration.alert.common.enumeration.ProcessingType;
-import com.synopsys.integration.util.Stringable;
+import com.synopsys.integration.alert.common.rest.model.AlertSerializableModel;
 
 // NotificationContentWrapper is not serializable, so this class cannot be serializable (and doesn't need to be)
-public class FilteredJobNotificationWrapper extends Stringable {
+//FIXME: This can't extend AlertSerializableModel
```

Comment:
What are we trying to achieve by making this AlertSerializableModel? It's kind of confusing that we made this serializable when we have comments saying we shouldn't and a new fixme that says we can't do this.

Reply:
This is being addressed in a followup PR by James when we fix paging. Right now this is required because we must return an AlertPagedModel which requires that the object its passing extends AlertSerializableModel. The plan is to address this still within 6.5.0.